docs/_source/tutorials-examples/population-synthesis/bbh_analysis.ipynb

```diff
@@ -603,7 +628,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With `TransientPopulation` class you can compute the efficiency of the transient population at a given metallicity, i.e. the number of merging DCO per unit star formation and visualize the results."
+    "With `TransientPopulation` class you can compute the efficiency of the transient population at a given metallicity, i.e. the number of merging DCO per unit star formation and visualize the results.\n",
+    "\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Underlying mass**\n",
+    "\n",
+    "Make sure you've calculated the underlying mass for the population, as done earlier in this tutorial.\n",
```

Comment:
May add that the rescaling with the underlying mass assumes, that in the not simulated part no transient events of this type occur. In case this happens, the values we get are strictly speaking only lower limits.